estate/security/ir.model.access.csv

```diff
@@ -2,4 +2,5 @@ id,name,model_id/id,group_id/id,perm_read,perm_write,perm_create,perm_unlink
 access_estate_property,estate_property,model_estate_property,base.group_user,1,1,1,1
 access_estate_property_type,estate_property_type,model_estate_type,base.group_user,1,1,1,1
 access_estate_property_tag,estate_property_tag,model_estate_tag,base.group_user,1,1,1,1
-access_estate_property_offer,estate_property_offer,model_estate_property_offer,base.group_user,1,1,1,1
```

Comment:
Try adding the missing EOL in the commit where the line was introduce.
You can typically do this using `git rebase -i` and `git push --force-with-lease`.